src/class-parsely.php

```diff
@@ -50,6 +50,39 @@ class Parsely {
 		'parsely_wipe_metadata_cache' => false,
 	);
 
+	/**
+	 * Declare post types that Parsely will process as posts
+	 *
+	 * @see https://www.parse.ly/help/integration/jsonld#distinguishing-between-posts-and-pages
+	 * @since 2.5.0
+	 * @var string[]
```

Comment:
```suggestion
	 * Declare post types that Parse.ly will process as "posts".
	 *
	 * @link https://www.parse.ly/help/integration/jsonld#distinguishing-between-posts-and-pages
	 *
	 * @since 2.5.0
	 * @var string[]
```